app/controllers/admin/participants_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
   private
 
     def load_participant
-      @participant = User.is_participant.find(params[:id])
+      @participant = User.is_ecf_participant.find(params[:id])
```

Comment:
Why this change? This should be all participants I think?

Reply:
NPQ will use this interface for manual validation

Reply:
oops